include/Webservices/ExecuteWorkflow.php

```diff
@@ -60,13 +60,17 @@ function cbwsExecuteWorkflow($workflow, $entities, $user) {
 	$workflow = reset($workflows);
 	foreach ($crmids as $crmid) {
 		$entityData = $entityCache->forId($crmid);
-		if ($workflow->evaluate($entityCache, $entityData->getId())) {
-			if (VTWorkflowManager::$ONCE == $workflow->executionCondition) {
-				$entity_id = vtws_getIdComponents($entityData->getId());
-				$entity_id = $entity_id[1];
-				$workflow->markAsCompletedForRecord($entity_id);
+		$modPrefix = $entityData->getModuleName(); // it return module from CRMIDs of selected records
+		$workflow_mod = $workflow->moduleName; // it return module of workflow
```

Comment:
this assignment is constant inside the loop, so it should be done outside the loop, not inside on every iteration
Please create a clean pull request, with no "merge master" and no issue cards and with this assignment outside the loop